pypeit/spectrographs/gtc_osiris.py

```diff
@@ -0,0 +1,337 @@
+"""
+Module for GTC OSIRIS specific methods.
+
+.. include:: ../include/links.rst
+"""
+import glob
+from pkg_resources import resource_filename
+
+import numpy as np
+
+from pypeit import msgs
+from pypeit import telescopes
+from pypeit.core import parse
+from pypeit.core import framematch
+from pypeit.spectrographs import spectrograph
+from pypeit.images import detector_container
+
+def flip_fits_slice(s: str) -> str:
+    return '[' + ','.join(s.strip('[]').split(',')[::-1]) + ']'
+
+
+class GTCOSIRISSpectrograph(spectrograph.Spectrograph):
+    """
+    Child of Spectrograph to handle GTC/OSIRIS specific code
+    """
+    ndet = 2
+    name = 'gtc_osiris'
+    telescope = telescopes.GTCTelescopePar()
+    camera = 'OSIRIS'
+    supported = False
+
+    def get_detector_par(self, hdu, det):
+        """
+        Return metadata for the selected detector.
+
+        Detector data from `here
+        <http://www.gtc.iac.es/instruments/osiris/>`__.
+
+        Args:
+            hdu (`astropy.io.fits.HDUList`_):
+                The open fits file with the raw image of interest.
+            det (:obj:`int`):
+                1-indexed detector number.
+
+        Returns:
+            :class:`~pypeit.images.detector_container.DetectorContainer`:
+            Object with the detector metadata.
+        """
+
+        binning = self.get_meta_value(self.get_headarr(hdu), 'binning')
+        datasec1 = hdu[1].header['datasec']
```

Comment:
It doesn't look like these are used.  If not, comment out or remove.

Reply:
Removed.